### DISCUSSION TITLE
Tiered membership nft with different voting power per tier

### DISCUSSION THREAD

**Discussion by ghost** [snapshot-labs/snapshot#2849] [Q&A] [answered]
I'm creating a tiered membership nft that will have voting power over a DAO. For voting I'd like to use snapshot. My question is, would it be possible to create a strategy where each tier has different voting power and if so does the metadata have to be onchain?
If this wouldnt be possible, would it be a work around to create a new nft contract for each tier and create a strategy around that?

**Comment by ChaituVR** [comment, marked as answer]
Hey yes, you can try a strategy like this https://snapshot.org/#/strategies?q=metadata if this doesn't work you can create a new strategy, make sure to follow 
https://docs.snapshot.org/strategies/create

**Reply by mseidlx** [reply]
@ChaituVR this link is unfortunately not working anymore - is there a new link for the strategy? Or sample code?

**Reply by metrox-eth** [reply]
Here is a working link:
https://snapshot.org/#/?type=strategies&q=met

**Comment by aitingting666** [comment]
你这个想法很棒
